Dockerfile

```diff
@@ -1,28 +1,39 @@
 # get official nodejs/npm binaries
-FROM node:22.9-slim as nodebuilder
+FROM node:22.9-alpine as nodebuilder
 WORKDIR /opt/app
 COPY package-lock.json ./
 COPY package.json ./
 RUN npm i
 
 # bundle install the gems for production
-FROM ruby:3.3.5 as rubybuilder
-RUN apt update -y \
-    && apt -y install nano \
-    && cp /usr/bin/nano /usr/local/bin/
+FROM ruby:3.3.5-alpine as rubybuilder
+RUN apk update && apk add --no-cache \
+    build-base \
+    postgresql-dev \
+    tzdata \
+    nano \
```

Comment:
I know we were installing this previously, but I'm pretty sure we don't need a text editor in the image. We're no longer copying it to /usr/local/bin so I think we can drop this.